### PR TITLE
FamilyMemberHistory Beta

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
FamilyMemberHistory Beta Read, Search, Create, Update

Also removed the if-match header documentation from QuestionnaireResponse PUT

Screen Shot 2021-06-03 at 12 40 19 PM
Screen Shot 2021-06-03 at 9 01 29 AM
Screen Shot 2021-06-03 at 12 15 46 PM
Screen Shot 2021-06-03 at 12 16 04 PM
Screen Shot 2021-06-03 at 12 16 08 PM
Screen Shot 2021-06-03 at 12 16 13 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/summary/family-member-history.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+<%= headers status: 200 %>
+<%= json(:r4_family_member_history_entry) %>
+
+<%= disclaimer %>
+
+## Create
+
+<%= beta_tag(action: true) %>
+
+Create new family member history.
```
Comment (RayGunY) [inline content/millennium/r4/summary/family-member-history.md:95]: Should this maybe be 'Create a new FamilyMemberHistory.'

I've seen other two word resources (FinancialTransaction, RelatedPerson, etc) do it that way, but I've also seen some like this is currently - so up to you. It just sticks out to me that Create is like this but Update and others refer to it as FamilyMemberHistory

**Review by MadhurRaj — COMMENTED** [review]
Can we get a screenshot for each action?

File: content/millennium/r4/summary/family-member-history.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+The following fields are returned if valued:
+
+* [FamilyMemberHistory id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+ 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for FamilyMemberHistory that meet supplied query parameters:
```
[the remark this answers is not in the document]
Comment (kathypickering) [reply content/millennium/r4/summary/family-member-history.md:26]: Yes

**Review by kathypickering — APPROVED** [review]

**Review by bh021790 — APPROVED** [review]

File: content/millennium/r4/summary/family-member-history.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Update
+
+<%= beta_tag(action: true) %>
+
+Update an FamilyMemberHistory.
```
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply content/millennium/r4/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/660/commits/8b441b3fc17a938a03c796812854104db997ead0

File: content/millennium/r4/summary/family-member-history.md (hunk 1 above)
Comment (sydneanschutz) [reply content/millennium/r4/summary/family-member-history.md]: I like it your way https://github.com/cerner/fhir.cerner.com/pull/660/commits/8b441b3fc17a938a03c796812854104db997ead0

File: content/millennium/r4/summary/family-member-history.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply content/millennium/r4/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/660/commits/8b441b3fc17a938a03c796812854104db997ead0

**Review by bh021790 — APPROVED** [review]

File: layouts/millennium/r4/summary.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<h1 id="security">Summary</h1>
+
+<p>Summary's summary</p>
```
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply layouts/millennium/r4/summary.html]: Summary category was removed: https://github.com/cerner/fhir.cerner.com/pull/660/commits/57b8731ef0883a3335016e5b43a97013f354d7af

**Review by RayGunY — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by sydneanschutz** [comment]
Validated on live
Screen Shot 2021-06-03 at 3 57 32 PM

**Pull request merged**